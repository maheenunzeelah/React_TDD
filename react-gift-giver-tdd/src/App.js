

function App() {
  return (
    <div className="App">
       <h1>Changes</h1>
       <h1>Changes</h1> 
    </div>
  );
}

export default App;
