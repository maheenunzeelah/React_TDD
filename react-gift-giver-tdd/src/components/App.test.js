import { render, screen } from '@testing-library/react';
import App from '../App';
import renderer from 'react-test-renderer'
describe("Testing",()=>{
  test("renders",()=>{
    // let app=renderer.create(<App />);
    // app=app.toJSON();
    const {container}=render(<App/>)
    expect(container).toMatchSnapshot();
  })
})