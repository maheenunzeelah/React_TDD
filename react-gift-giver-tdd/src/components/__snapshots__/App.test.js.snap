// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing renders 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Changes
    </h1>
  </div>
</div>
`;
